File: Pratica/beecrowd/javascript/1010.js
const space = " "
    
const [cod1, qtd1, valor1] = lines[0].split(space)
const [cod2, qtd2, valor2] = lines[1].split(space)

const res = (qtd1 * valor1) + (qtd2 * valor2);

console.log(`VALOR A PAGAR: R$ ${res.toFixed(2)}`);

File: Pratica/beecrowd/java/1010.java
import java.util.Scanner;

public class 1010{
    public static void main(String[] args){
        
        Scanner sc = new Scanner(System.in);
        
        int cod1 = sc.nextInt();
        int pec1 = sc.nextInt();
        float val1 = sc.nextFloat();
        
        int cod2 = sc.nextInt();
        int pec2 = sc.nextInt();
        float val2 = sc.nextFloat();
        
        float total = (pec1 * val1)+(pec2*val2);
        
        System.out.printf("VALOR A PAGAR: R$ %.2f\n", total);
        
        sc.close();
    }
}